tests/integration/test_aws/data/test_cases/regions_test_module/cases_service_regions.yaml

```diff
@@ -0,0 +1,32 @@
+- name: inspector_region_with_data
+  description: Inspector regions configurations
+  configuration_parameters:
+    SERVICE_TYPE: inspector
+    REGIONS: us-east-1
+  metadata:
+    service_type: inspector
+    only_logs_after: 2023-JAN-30
+    regions: us-east-1
+    expected_results: 4
+
+- name: inspector_regions_with_data
+  description: Inspector regions configurations
+  configuration_parameters:
+    SERVICE_TYPE: inspector
+    REGIONS: us-east-1,us-east-2
+  metadata:
+    service_type: inspector
+    only_logs_after: 2023-JAN-30
+    regions: us-east-1,us-east-2
+    expected_results: 4
```

Comment:
Just a small question that comes to mind: does `us-east-2` region have no events? or why is the same number of events expected for region `us-east-1` as for `us-east-1` +  `us-east-2`?

Reply:
There are 4 events in both regions. But, the test is expecting to find one message for each region set.

```python
    wazuh_log_monitor.start(
            timeout=T_20,
            callback=event_monitor.callback_detect_inspector_event_processed(expected_results),
            error_message='The AWS module did not process the expected number of events',
            accum_results=len(regions_list)
        ).result()
```